src/objects.rs

```diff
@@ -254,6 +259,18 @@ pub struct Relation {
     pub tags: Tags,
     /// Members of the relation.
     pub refs: Vec<Ref>,
+    /// The info of the relation.
+    pub info: Option<Info>,
+}
+
+/// An Info object for storing metadate about a Node, Way or Relation.
+#[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Clone, Hash, Serialize, Deserialize)]
+pub struct Info {
+    /// The timestamp when the object was introduced.
+    pub timestamp: Option<NaiveDateTime>,
```

Comment:
I'm a bit worried by this:
 * there is 2 options inside each ones, that makes it difficult to use
 * the is_visible is obviously true if there is no info
 * all this option thing will use a non negligible amount of memory.

I don't have yet any solution to all of this.

Reply:
Mh yeah I'm not happy with the double Option either.

I think it makes sense to keep the Info object, as the osm protobuf definition provides some more fields like `changeset` and `uid`, so one might want to extend it in the future.

Not sure wether one could find an Info without a timestamp in the wild. I certainly didn't find one. Maybe it would make sense to make the timestamp non-optional?

Reply:
Sorry, forgot about this PR.

Yeah, I think we can:
* make timestamp non optional, outputing 0 in the rare case when we have no info
* `Option<NonZeroI64>`, with some appropriate getters for ease of use. A timestamp of 0 is impossible as OSM didn't exist in 1970.

Reply:
My OSM related project is on ice, so I won't be able to help further with this PR, sorry. Good luck!